### DISCUSSION TITLE
Videos from Pixel 9 Pro longer than 30 sec are not uploaded

### DISCUSSION THREAD

**Discussion by msch4ever** [immich-app/immich#14820] [Q&A]
### The bug

I changed from samsung to Pixel 9 Pro. I quickly saw a bug. Longer videos are not saved in Immich.
Backup goes through all assets that can be uploaded and then the "Remainder" stays the same despite upload indicates that videos are being uploaded.
Happenes after I switched to Pixel. Already 3 major versions of Immich, but I don't think it applies to specific immich version.

### The OS that Immich Server is running on

OMV Debian

### Version of Immich Server

1.22.3

### Version of Immich Mobile App

1.123.0 build.172

### Platform with the issue

- [X] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Default contents
look at your reference documentation
```

### Your .env content

```Shell
Default contents
look at your reference documentation
```

### Reproduction steps

1. Get pixel 9 Pro
2. record at least two videos that is around 200MB in size in mp4 format
3. Press start backup

Expected:
- new videos uploaded on server
Actual:
- backup is looping infinitely trying to upload video assets, progress bar indicates that assets are processed, but Remainder is not decremented
![photo_1_2024-12-20_11-59-49](https://github.com/user-attachments/assets/9664efd4-b097-45c1-890d-b88ced266387)
![photo_2_2024-12-20_11-59-49](https://github.com/user-attachments/assets/f5a48dd0-2bf3-4242-97f2-a430f6cb7969)

**Comment by bo0tzz** [comment]
Are you accessing Immich directly on ip:port, or using something inbetween such as a reverse proxy?

**Reply by msch4ever** [reply]
I have set up switching between local network and public domain endpoint and not sync works. I did not have this problems before though when I had similar setup while using samsung phone. Is it just coincidence and cloudflare introduced this constraint recently and at the same day I switched to Pixel?:)

**Comment by msch4ever** [comment]
My App is exposed with Cloudflare secure tunnel.
I can try doing the backup via local network.

**Reply by bo0tzz** [reply]
Cloudflare has a size limit on uploads
